app/2.0/docs/devguide/dom-template.md

```diff
@@ -10,7 +10,7 @@ By default, adding a DOM template to an element causes Polymer to create a shado
 
 The DOM template is also processed to enable data binding and declarative event handlers.
 
-## DOM template overview
+## Specify a DOM template
 
 To specify an element's DOM template:
 
```

Comment:
There's really a step (3) that goes along with 1 & 2, which is

> 3. Add a `static get is() { return 'element-name'; }` getter to your element's class that identifies the `dom-module` id to get the template from.

However, it's probably worth noting that we now support 3 ways of specifying a template:

1. `static get is()` that returns id of a `dom-module` containing a `<template>` (basically, the normal legacy way, which should be the most efficient way to package a template using HTMLImports)
1. `static get template()` that returns an `HTMLTemplateElement` reference
1. `static get template()` that returns a string (which Polymer will parse into a new HTMLTemplateElement)

And note that if you're not using method (1), there's now no reason to include the `static get is()` -- its only purpose is to identify the dom-module containing a template, and that is no longer the compulsory way to get a template.

There's some detail in the 3rd bullet in the bulleted list under https://github.com/Polymer/polymer/blob/master/README.md#20-es6-class-based-syntax

---

Sorry, I see you cover (3) below... however it may make sense to lay out the 3 ways up front and then describe each one with an example maybe?